README.md

```diff
@@ -1,138 +1,248 @@
+Trino Python client
+===================
+A [Trino](https://trino.io/) client for the [Python](https://www.python.org/) programming language.
+
 [![Build Status](https://github.com/trinodb/trino-python-client/workflows/ci/badge.svg)](https://github.com/trinodb/trino-python-client/actions?query=workflow%3Aci+event%3Apush+branch%3Amaster)
 [![Trino Slack](https://img.shields.io/static/v1?logo=slack&logoColor=959DA5&label=Slack&labelColor=333a41&message=join%20conversation&color=3AC358)](https://trino.io/slack.html)
 [![Trino: The Definitive Guide book download](https://img.shields.io/badge/Trino%3A%20The%20Definitive%20Guide-download-brightgreen)](https://www.starburst.io/info/oreilly-trino-guide/)
 
-# Introduction
-
-This package provides a client interface to query [Trino](https://trino.io/)
-a distributed SQL engine. It supports Python>=3.6 and pypy.
-
-# Installation
+## Usage
+### The Python Database API (DBAPI)
 
+**Installation**
 ```
 $ pip install trino
 ```
 
-# Quick Start
+**Quick Start**
 
 Use the DBAPI interface to query Trino:
 
 ```python
-import trino
-conn = trino.dbapi.connect(
-    host='localhost',
-    port=8080,
-    user='the-user',
-    catalog='the-catalog',
-    schema='the-schema',
+from trino.dbapi import connect
+
+conn = connect(
+    host="<host>",
+    port=<port>,
+    user="<username>",
+    catalog="<catalog>",
+    schema="<schema>",
 )
 cur = conn.cursor()
-cur.execute('SELECT * FROM system.runtime.nodes')
+cur.execute("SELECT * FROM system.runtime.nodes")
 rows = cur.fetchall()
 ```
 
-This will query the `system.runtime.nodes` system tables that shows the nodes
-in the Trino cluster.
+This will query the `system.runtime.nodes` system tables that shows the nodes in the Trino cluster.
```

Comment:
I would recommend extracting a commit for column width changes.

Reply:
I don't think it change the column width because there are some lines event longer, e.g
https://github.com/trinodb/trino-python-client/blob/65506e8f1c90ea8908576d43037be46f53c8604a/README.md?plain=1#L199

Reply:
I suggested separating into two commits. 
* Add SQLArchemy example in README
* Change column width in README

Reply:
@ebyhr the column width config for now is `120`, right?
Let me re-format the code to match this rule.

Reply:
@ebyhr For line contain link, it's quite long (for example this one is 162 chars) but can't be break in to 2 lines.
Is it ok to keep it in one line? 
![image](https://user-images.githubusercontent.com/6848311/153702956-61bb9f70-a574-4b94-ab47-25675db0744a.png)

Reply:
The 120 is for Python files, not markdown. 

> Is it ok to keep it in one line?

Yes, it's ok.

Reply:
@ebyhr yes, I known. But I don't find any docs/config talking about markdown convention.

Reply:
Resolved as personal discuss.